Dockerfile

```diff
@@ -1,16 +1,18 @@
-FROM node:16-bullseye as base
+FROM alpine as base
```

Comment:
I think we can use "node:lts-alpine" image here and don't install anything later